### ISSUE TITLE
Improve the useabilty of Selection Sort program

### ISSUE THREAD

**Issue by Zircoz** [Zircoz/Sorting-Algorithms-in-Python#16]
The code written in #12 takes an array of numbers seperated by whitespaces. If anyone can make it more 'user-friendly' it will be great!

Please explain how your changes make the existing code better.

**Comment by simonbukin** [comment]
The selection_sort function itself looks functionally correct to me, what exactly do you mean by "user-friendly"? It looks like the function just takes a list as input, so the input that goes into it needs to be changed I'm assuming?

**Comment by Zircoz** [comment]
Yeah, I would image an easier way of entering the numbers, like one by one. Or something you can think of.

**Comment by simonbukin** [comment]
Maybe it would be a good idea to implement testing? Something like pytest would let us make sure all the algorithms in the repo are functional on a variety of input. 

I'm still not entirely sure what you mean by adding easier number input, but it looks like the file was changed in #17 . Is that more what you were looking for?

**Comment by dwijenchawra** [comment]
I'd like to work on this!

**Comment by anirudhbagri** [comment]
Can you check #23, if this is what we wanted..

**Comment by dwijenchawra** [comment]
I have already resolved this issue... `https://github.com/Zircoz/Sorting-Algorithms-in-Python/pull/21#issue-327618293`

**Comment by Zircoz** [comment]
Both solutions are pretty similar, @dwijenchawra & @anirudhbagri .
I will merge Chawra's PR when it is approved based on first come first serve basis.

**Comment by Zircoz** [comment]
Yes, @simonbcodes testing implementation sounds good to me and it can be a separate issue of its own but I am not familiar with it yet. So, I will try it in local sandbox and get back to this when I am ready and have enough time.

**Comment by Zircoz** [comment]
> 
> 
> Both solutions are pretty similar, @dwijenchawra & @anirudhbagri .
> I will merge Chawra's PR when it is approved based on first come first serve basis.

I just discovered the #20 and it provides better usability while giving almost similar way of ending array as @dwijenchawra and @anirudhbagri 's. So, it is being merged and thanks to everyone else who contributed to this issue. Feel free to even improve it even further in any direction.
